src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -27,12 +27,12 @@
         implements Runnable {
     private static final String CLASSPATH_SEPARATOR = ":";
 
+    @Spec
```

Comment:
Spec annotation moved to another field. Was the previous one incorrect?

Reply:
Yes, it was mistakenly changed in another commit.